runtime/CSharp/src/Atn/ConflictInfo.cs

```diff
@@ -1,80 +0,0 @@
-/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
```

Comment:
Is it possible someone is using this from the outside world?

Reply:
Good question, but I don't think so. There is no documentation about this class, and there is no mention in other runtimes about such entity. The purpose of `ConflictInfo` is unclear.